python/Ganga/GPIDev/Lib/File/File.py

```diff
@@ -294,6 +296,20 @@ def isExecutable(self):
         file are checked"""
         return self.executable or is_executable(expandfilename(self.name))
 
+    def removeAssociatedFiles(self):
+        """ Remove the files in the associated file list"""
+        for entry in list(self.associated_files):
```

Comment:
Should the list in `self.associated_files` not be replaced with an empty list (or `None`) at the end of this loop?